index.js

```diff
@@ -115,6 +115,13 @@ Client.prototype.sync = function (callback) {
   if (typeof callback !== 'function') throw new Error('sync missing callback parameter')
 
   if (!self.state.properties) self.state.properties = {}
+  if ((self.state.reconcileStamp === null) || (isNaN(self.state.reconcileStamp))) {
+    if (self.state.properties.reconcileDate) self.state.reconcileStamp = new Date(self.state.properties.reconcileDate).getTime()
+    if (self.state.properties.days) self.state.reconcileStamp = self.state.properties.days * msecs.day
```

Comment:
do you actually want `self.state.properties.days` to override `self.state.properties.reconcileDate`, or should this be `else if` instead of `if`?